src/main/resources/plugin.yml

```diff
@@ -6,3 +6,21 @@ prefix: Warpstones
 authors: [Atrius]
 description: Warp stones for survival
 website: https://atrius.xyz
+commands:
+  waystones:
+    description: Displays plugin info and usage
+    usage: /waystones [command]
+    aliases: [ws]
+permissions:
+  waystones.*:
+    description: Gives access to all Waystones commands
+    children:
+      waystones.reload: true
```

Comment:
I don't believe this is a necessary permission based off the direction the config branch is going.